test-data/unit/check-enum.test

```diff
@@ -6,10 +6,37 @@ class Medal(Enum):
     gold = 1
     silver = 2
     bronze = 3
+reveal_type(Medal.bronze)  # E: Revealed type is '__main__.Medal'
 m = Medal.gold
-m = 1
-[out]
-main:7: error: Incompatible types in assignment (expression has type "int", variable has type "Medal")
+m = 1  # E: Incompatible types in assignment (expression has type "int", variable has type "Medal")
+
+[case testEnumFromEnumMetaBasics]
+from enum import EnumMeta
+class Medal(metaclass=EnumMeta):
+    gold = 1
+    silver = "hello"
+    bronze = None
+reveal_type(Medal.bronze)  # E: Revealed type is '__main__.Medal'
+m = Medal.gold
+m = 1  # E: Incompatible types in assignment (expression has type "int", variable has type "Medal")
+
+[case testEnumFromEnumMetaSubclass]
+from enum import EnumMeta
+class Achievement(metaclass=EnumMeta): pass
+class Medal(Achievement):
+    gold = 1
+    silver = "hello"
+    bronze = None
+reveal_type(Medal.bronze)  # E: Revealed type is '__main__.Medal'
+m = Medal.gold
+m = 1  # E: Incompatible types inassignment (expression has type "int", variable has type "Medal")
```

Comment:
Needs a space between "in" and "assignment"

Reply:
Yep. Sorry about that.